setup.py

```diff
@@ -123,6 +114,7 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
         "Operating System :: Microsoft :: Windows",
```

Comment:
We no longer "officially" support/test Windows.

Reply:
Removed